com.pot_app.pot.yml

```diff
@@ -0,0 +1,177 @@
+id: com.pot_app.pot
```

Comment:
pot-app.com doesn't seem to resolve. It has to be a real domain under the control of the project.

Reply:
Hmm, But that's what the project's homepage is on GitHub

Reply:
You should contact them about fixing that.